Resources.md

```diff
@@ -4,42 +4,38 @@ For example in Project there's some really minor steps while the first point of
 I've outlined below a few suggestions also indicating what I think are prerequisites #pq or there might be an overlap -> 
 It's just an example to explain what I think might be a more even granularity rather then a comment on content. 
 
-### Project
-
- * Determining appropriate CLEX project code,
- * Contact project coordinator (CI or delegate)
- * Request to join a project through mancini  #pq
- * Determining CI of project, #pq? (I'm thinking either via mancini, so potentially redundant or via wiki since mancini might not have enough "scientific" info)
-
-### Resource request
+### Setup
```

Comment:
I don't think that's the right word. Just want to say: what to do before using any resource.
I've merged Project and Resource request together as it would make a better training block as one.